packages/flame/lib/src/components/core/component.dart

```diff
@@ -974,9 +974,10 @@ class Component {
   /// Returns a [TextPaint] object with the [debugColor] set as color for the
   /// text.
   TextPaint get debugTextPaint {
+    final zoom = CameraComponent.currentCamera?.viewfinder.zoom ?? 1.0;
```

Comment:
It would be good to take the scale of the viewport into account too

Reply:
Are you talking about `camera.viewport.size`?

Reply:
Ah, apparently only the `FixedResolutionViewport` exposes the scale, so we can skip that for now.